site/docs/v1/tech/lambdas/sony-playstation-network-reconcile.adoc

```diff
@@ -0,0 +1,63 @@
+---
+layout: doc
+title: Sony Playstation Network Reconcile
+description: An overview of the Sony Playstation Network Reconcile lambda
+---
+
+:sectnumlevels: 0
+
+== Sony Playstation Network Reconcile lambda
+
+When an Sony Playstation Network identity provider is used to complete a federated login request FusionAuth will use the email address to reconcile user. You may optionally utilize a lambda to customize the user and user registration during this authentication event.
+
+When you create a new lambda using the FusionAuth UI we will provide you an empty function for you to implement.
+
+If you are using the API to create the lambda you will need to ensure your function has the following signature:
+
+[source,javascript]
+----
+function reconcile(user, registration, idToken) {
+  // Lambda code goes here
+}
+----
+
+This lambda must contain a function named `reconcile` that takes three parameters. The parameters that the lambda is passed are:
+
+* `user` - the FusionAuth User object
+* `registration` - the FusionAuth UserRegistration object
+* `idToken` - the JSON payload returned by the Sony Playstation Network Token Info API
```

Comment:
The internal service is naming this `userInfoJson`, we can name it whatever we want. However, it is not an `idToken`. The OIDC lambda calls it a `jwt`, it is just the JSON response from the Userinfo endpoint. 

We need to correct the Lambda Type enum as well. 
https://github.com/FusionAuth/fusionauth-java-client/blob/39ed46a3380816695d987988f676dc37df51abfe/src/main/java/io/fusionauth/domain/LambdaType.java#L264